MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1575,6 +1575,7 @@
     _features.shaderStorageBufferArrayDynamicIndexing = true;
     _features.shaderClipDistance = true;
     _features.shaderInt16 = true;
+	_features.shaderInt64 = mslVersionIsAtLeast(MTLLanguageVersion2_2);
```

Comment:
I think we should only enable this for MSL 2.3. That's the first version that allows loading and storing 64-bit integers without a bitcast. If we want to support this on 2.2, we'll need to have SPIRV-Cross insert those bitcasts, and somehow I don't think H-K is amenable to that.

We may also want to limit this to particular GPU families. According to the Metal Feature Set Tables, only Apple families 3 through 7 support 64-bit integers fully. It might work on Mac families 1 and 2; needs testing.